CMakeLists.txt

```diff
@@ -16,11 +17,13 @@ add_executable(picoprobe
         src/probe.c
         src/cdc_uart.c
         src/get_serial.c
+        src/ws2812.c
 )
 
+# This stanza isn't working -- why not?
+# Plan is to include QTPy WS2812-specific Cmakle cmds here
```

Comment:
I think your `add_definitions(-DUSE_QPTY_SWD` _only_ adds definitions visible to the compiler, whereas you need to use some other syntax if you want something that's visible/usable by cmake itself?